dodo.py

```diff
@@ -353,7 +390,7 @@ def common_update(root, branch):
             'cd %s/template && git checkout master' % root,
             'cd %s/template && git fetch --all --depth=2' % root,
             'cd %s/template && git merge --allow-unrelated-histories -X theirs --squash project/%s' % (root, branch),
-            'cd %s/template && composer update --no-interaction' % root
+            'cd %s/template && composer update --ignore-platform-reqs --no-interaction' % root
```

Comment:
That argument sounds like you're asking for our customers to have trouble on production.

Reply:
That's only run while building the template repo locally (on one of our computers).  Since the resulting container for Magento is still built on 7.1 and runs without issue it doesn't seem to cause a problem.  It's just allowing it to run `composer install` on any PHP version for the initial creation.